python/pyspark/sql/streaming/query.py

```diff
@@ -165,6 +165,11 @@ def explain(self, extended: bool = False) -> None:
 
         Examples
```

Comment:
Should also probably add examples to the API that do not have examples, e.g., `stop`.

Reply:
or `processAllAvailable`

Reply:
Addressed in https://github.com/apache/spark/pull/37482/commits/7d890d68fc82d860d6783976f30673d11d7ef4c9